### DISCUSSION TITLE
Modify Ace Cargo Mission.sqf

### DISCUSSION THREAD

**Discussion by Raz0rLegend** [Vdauphin/HeartsAndMinds#1242] [Question and answer] [answered]
Hi Dauphin ! 

I saw the format to the Mission.sqf changed between the master and the master_daily. In this last, i can't modify the ace cargo. And for example, i can put only one container inside the hemtt container againt 5 inside your master version.
I tried to put the code inside your new mission.sqf, but nothing changed.
What's the thing i did badly ? 

https://github.com/Vdauphin/HeartsAndMinds/blob/cc81e249f5591ecf890d41d35679f7fbe3cfcd1f/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L538
 ```
btc_log_main_cc = [
    "Helicopter", 6,
    "Ship", 3,
    "Tank", 5,
    "Wheeled_APC", 5,
    "Truck", 10,
    "Truck_F", 10,
    "Motorcycle", 1,
    "Car", 3
];
btc_log_main_rc = [
    "Items_base_F", 2,
    "ReammoBox_F", 2,
    "thingX", 3,
    "StaticWeapon", 3,
    "Strategic", 2,
    "Motorcycle", 3,
    "Land_BarGate_F", 3,
    "HBarrier_base_F", 5,
    "Land_BagFence_Long_F", 3,
    "Wall_F", 5,
    "BagBunker_base_F", 5,
    "Wheeled_APC", 50,
    "Tank", 75,
    "Truck", 50,
    "Truck_F", 50,
    "Ship", 50,
    "Helicopter", 9999,
    "Car", 35,
    "Lamps_base_F", 2
];
btc_log_def_cc = [
    "Land_CargoBox_V1_F", 0,
    btc_supplies_cargo, 20,
    btc_fob_mat, 0,
    "Land_Cargo20_military_green_F", 20,
    "Land_Cargo40_military_green_F", 40
];
btc_log_def_rc = [
    "Land_BagBunker_Small_F", 5,
    "Land_CargoBox_V1_F", 9999,
    btc_supplies_cargo, 8,
    btc_fob_mat, 8,
    "Land_Cargo20_military_green_F", 8,
    "B_Slingload_01_Fuel_F", 20,
    "Land_Pod_Heli_Transport_04_medevac_black_F", 8
];
```
Thanks in advance.
Cheers

**Comment by Raz0rLegend** [comment]
Ok, first of all, I understand there is no more get_cc and get_rc fnc.

**Comment by Vdauphin** [comment, marked as answer]
Hello,
> I tried to put the code inside your new mission.sqf, but nothing changed.

You can't put stuff and hope it will works. You need to understand the new implementation.

> What's the thing i did badly ?

Nothing. The master_daily branch contain major changes. **ANY code from master will NOT work in master_daily.**

> In this last, i can't modify the ace cargo.

Now Hearts and Minds doesn't tweak ACE cargo of objects that have already a defined ACE cargo property.
For gameplay reason we just define the space cargo of FOB and HEMTT: https://github.com/Vdauphin/HeartsAndMinds/blob/cc81e249f5591ecf890d41d35679f7fbe3cfcd1f/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/eh/server.sqf#L135-L145

Cheers

**Reply by Raz0rLegend** [reply]
Perfect ! I searched this #cargo one hour and I did not think about the EventHandlers. 
Many thanks again !
